docs/build-on-linea/quickstart/deploy-smart-contract/cookbook.md

```diff
@@ -68,248 +75,277 @@ Choose **No-Code Deploy** on select (usually simpler) smart contracts on [Cookbo
-  <div class="img-medium">
-      <img
-      src="/img/article_images/Build_on_Linea/Quickstart/Deploy_a_smart_contract/Cookbook/Linea_deploy_smart_contract_Cookbook_8.png"
-      alt="Cookbook Remix Featured Plugin"
-    />
-  </div>
-</div>
+1. Compile and deploy the smart contract as described in [the Remix instructions](./remix.md).
 
-Alternatively, search Cookbook and select **Activate** in the Remix Plugin Manager.
+## Deploy your smart contract with Hardhat
 
-<div class="center-container">
-  <div class="img-small">
-      <img
-      src="/img/article_images/Build_on_Linea/Quickstart/Deploy_a_smart_contract/Cookbook/Linea_deploy_smart_contract_Cookbook_9.png"
-      alt="Cookbook Remix Add Plugin"
-    />
-  </div>
-</div>
+After finding the smart contract or protocol you want to work with in [Cookbook](https://www.cookbook.dev/?utm=lineadocs),
+select the **Download Source** option and select **Hardhat** to download the contract boilerplate. For
+this example, we'll use [Cookbook's Simple ERC-20 Token Smart Contract](https://www.cookbook.dev/contracts/simple-token?utm=lineadocs).
 
-Search for any protocol or smart contract and click the search result to import the smart contract code into Remix.
 
-Cookbook's AI solidity co-pilot, ChefGPT, is available within the Remix plugin to answer questions about Linea, Solidity, or the smart contract you're working with.
+### Compile the smart contract
 
-<div class="center-container">
-  <div class="img-small">
-      <img
-      src="/img/article_images/Build_on_Linea/Quickstart/Deploy_a_smart_contract/Cookbook/Linea_deploy_smart_contract_Cookbook_10.png"
-      alt="Cookbook Remix Search"
-    />
-  </div>
-</div>
-
-Compile and deploy the smart contract as described in **Method 1** above.
-
-## Deploy your Smart Contract to Linea with Hardhat
-
-After finding the smart contract or protocol you want to work with in [Cookbook](https://www.cookbook.dev/?utm=lineadocs), select the **Download Source** option and select **Hardhat** to download the contract boilerplate. For this guide, we'll use [Cookbook's Simple ERC-20 Token Smart Contract](https://www.cookbook.dev/contracts/simple-token?utm=lineadocs).
-
-To install the required packages and dependencies, run
+In the project folder, install the required packages and dependencies:
 
 ```
 npm install
 ```
 
-To compile your smart contract, run
+Then, compile the smart contract:
 
 ```
 npx hardhat compile
 ```
 
-Add arguments to the `constructorArgs` array in the `deploy.js` file in the `scripts` folder and save. If you do not need any arguments please leave the array empty.
+Add arguments to the `constructorArgs` array in the `deploy.js` file in the `scripts` folder and save. If you
+do not need any arguments, leave the array empty.
 
-### Deploy To Linea
+### Deploy the smart contract
 
 <Tabs>
-  <TabItem value="Deploy To Linea Goerli (Testnet)" label="Testnet" default>
+  <TabItem value="Goerli" label="Goerli" default>
```

Comment:
I'd recommend we go with "Linea Goerli" for clarity. This was suggested by the Linea team and I've implemented it in other places